subsys/net/lib/download_client/src/download_client.c

```diff
@@ -134,7 +128,7 @@ static int set_recv_socket_timeout(int fd, int timeout_ms)
 		.tv_usec = (timeout_ms % 1000) * 1000,
 	};
 
-	err = setsockopt(fd, SOL_SOCKET, SO_RCVTIMEO, &timeo, sizeof(timeo));
+	err = zsock_setsockopt(fd, SOL_SOCKET, SO_RCVTIMEO, &timeo, sizeof(timeo));
```

Comment:
This is fine in principle, but we should try to keep this aligned throughout the codebase, at least in `sdk-nrf`. It would be a bit messy if some libraries used `zsock_` and others the POSIX names; what guideline do we set for ourselves? Is it up to personal preference which one to use?